example/string_generator.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+"""
+CherryPy String Generator Service.
+
+This is the example random string generator service, slightly
+modified for swagger to not use session support.
+"""
+import random
+import string
+import os
+import cherrypy
+
+STRING_FILE = 'string.txt'
+
+
+@cherrypy.expose
+class StringGeneratorWebService(object):
+    """String Generator Class."""
+
+    @cherrypy.tools.json_out()
+    # pylint: disable=no-self-use
```

Comment:
I believe you could [apply `@staticmethod` instead](http://docs.quantifiedcode.com/python-anti-patterns/correctness/method_could_be_a_function.html#best-practices), and drop `self`.